testsuite/MDAnalysisTests/analysis/test_diffusionmap.py

```diff
@@ -69,6 +74,13 @@ def test_dist_weights(u):
                                 [.707, -.707, 0, 0]]), 2)
 
 
+def test_distvalues_ag_universe(u, ag):
+    dist_universe = diffusionmap.DistanceMatrix(u, select='backbone').run()
+    dist_ag = diffusionmap.DistanceMatrix(ag).run()
```

Comment:
Honestly as much as I like fixtures I'm not sure it's worth building a module-level fixture just for the one test, particularly when this can just be `u.select_atoms('backbone')`